app/src/main/res/layout/feedback_item.xml

```diff
@@ -48,7 +48,7 @@
       android:breakStrategy="simple"
       android:fontFamily="sans-serif"
       android:text="@{htmlContent}"
-      android:textColor="@color/oppia_primary_text"
+      android:textColor="@color/question_player_activity_textview_highlighted_color"
```

Comment:
This naming is incorrect because this file is getting used in exploration player too.

Name something like 

`feedback_item_text_view_primary_color`

Reply:
Ditto for other files too.